src/promptflow/promptflow/executor/flow_executor.py

```diff
@@ -951,11 +967,17 @@ def _apply_inputs_mapping_for_all_lines(
         }]
         """
         if inputs_mapping is None:
-            # This exception should not happen since we should use default inputs_mapping if not provided.
-            raise NoneInputsMappingIsNotSupported("Inputs mapping is None.")
+            # This exception should not happen since we will use default inputs_mapping for None input.
+            raise NoneInputsMappingIsNotSupported(
+                message_format="Inputs mapping is None. "
+                "Please set one inputs mapping or use default inputs mapping in flow_executor."
```

Comment:
This is a system error, right? Can the user understand this troubleshoot message?

Reply:
The troubleshoot message is intended to be read by partner team such as PRS.